src/layouts/components/GroupChainStatsCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import ChainStatsCard from "@components/commons/ChainStatsCard";
+import Container from "@components/commons/Container";
+
+function GroupChainStatsCard() {
```

Comment:
any reason to have this component in `.../layouts/components`?

Reply:
placed in the wrong components folder have updated to `.../src/components`